VideoLocker/src/main/java/org/edx/mobile/util/images/TopAnchorWidthFillTransformation.java

```diff
@@ -0,0 +1,49 @@
+package org.edx.mobile.util.images;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.os.Build;
+
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.resource.bitmap.BitmapTransformation;
+
+/**
+ * Created by miankhalid on 9/9/15.
+ */
+public class TopAnchorWidthFillTransformation extends BitmapTransformation {
+    private final Paint paint = new Paint(Paint.FILTER_BITMAP_FLAG);
+
+    public TopAnchorWidthFillTransformation(Context context) {
+        super(context);
+    }
+
+    @Override
+    protected Bitmap transform(BitmapPool pool, Bitmap toTransform,
+                               int outWidth, int outHeight) {
+        final int width = toTransform.getWidth();
+        final float widthRatio = outWidth / (float) width;
+        if (outWidth > width) {
```

Comment:
This was not done properly in d7c556ebca0371e9042daa732989da6da765fb27; see my [comment][] there.

[comment]: https://github.com/edx/edx-app-android/commit/d7c556ebca0371e9042daa732989da6da765fb27#commitcomment-13161196